### PR TITLE
Printing Support moved to AtCore

### PR DESCRIPTION
**Pull request by sithlord48**
All print functions are in AtCore now. 
Protocol Layer is removed.

### CODE REVIEW

**Review by patrickelectric — COMMENTED** [review]
Please, use a title and after a text block in the commit message.

File: src/atcore.h [hunk 1]
```diff
@@ -17,9 +17,25 @@ Q_OBJECT
...
+signals:
+    /**
+     * @brief Emit signal when message is received from the printer
+     *
+     * @param msg : Message
+     */
+    void receivedMessage(const QByteArray& msg);
 private:
     void findFirmware(const QByteArray& message);
+    void newMessage(const QByteArray& msg);
```
Comment (patrickelectric) [inline src/atcore.h]: Be consistent with argument name, use message.

File: testclient/mainwindow.cpp [hunk 2]
```diff
@@ -97,7 +97,8 @@ void MainWindow::checkPushedCommands(QByteArray bmsg)
  * of serial ports
  */
 void MainWindow::locateSerialPort()
-{   QStringList ports;
+{
+    QStringList ports;
```
Comment (patrickelectric) [inline testclient/mainwindow.cpp:101]: This is a style correction, need to be in other commit.

Comment (patrickelectric) [reply testclient/mainwindow.cpp:101]: But Let it in this way, I will create a astyle script later.

**Review by patrickelectric — CHANGES_REQUESTED** [review]

File: src/atcore.h (hunk 1 above)
Comment (sithlord48) [reply src/atcore.h]: Changed argument name from msg to message

**Review by patrickelectric — APPROVED** [review]

**Comment by patrickelectric** [comment]
You can merge it

**Comment by sithlord48** [comment]
:heavy_check_mark: merged into kde/atcore

**Pull request closed without merging**